### DISCUSSION TITLE
How to get notified when object moves from zone B to Zone A with mqtt

### DISCUSSION THREAD

**Discussion by ronbruins** [blakeblackshear/frigate#14194] [Ask A Question] [answered]
I have frigate setup and use mqtt to get notifications. However I am still a bit struggling with the direction of movement and the before/after current/entered zones.

So if I want to make an automatic that will notify me when a cat moves from Zone B to A but not when it moves from A to B

This is to warn when there is a cat at the door_zone, coming from the terrace_zone. but no need to be notified when it goes from the door to the terrace or any other zones for that matter.

**Comment by NickM-27** [comment, marked as answer]
entered_zones contains a chronological list of zones entered. So to only be notified once the basic logic would be `before.entered_zones = ["terrace_zone"] AND after.entered_zones = ["terrace_zone", "door_zone"]`

**Reply by ronbruins** [reply]
Ah, so the zone where the object comes from is still mentioned in the `after.entered_zones`

**Comment by ronbruins** [comment]
I am copy pasting a lot from [https://github.com/blakeblackshear/frigate/discussions/2898](url)
and for the notification part, what is the difference between the clip url with/without the notification in the url?:
```
video: >-
    HASS_URL_BASE/api/{{id}}/{{camera}}/clip.mp4
     clickAction: >-
     http://HASS_URL_BASE/api/frigate/notifications/{{id}}/{{camera}}/clip.mp4
```

**Reply by NickM-27** [reply]
notification uses the integration api, otherwise you are using the frigate api and need to make sure the device has access to that directly.

**Comment by ronbruins** [comment]
Thanks for the quick reply!
